redis/server.hh

```diff
@@ -10,7 +10,7 @@
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
  *
- * Scylla is distributed in the hope that it will be useful,
+ * Scylla is seastar::sharded in the hope that it will be useful,
```

Comment:
Ooops :-) This will have to be fixed back :-)

Reply:
Yes please. Although the comment is still correct.